### PR TITLE
KUSD mint cap

### PR DESCRIPTION
**Pull request by arthurka-o**

### CODE REVIEW

File: packages/contracts/test/BorrowerOperationsTest.js [hunk 1]
```diff
@@ -4668,11 +4711,69 @@ contract("BorrowerOperations", async accounts => {
         B,
         { from: B }
       );
-
       // B attempts to repay all his debt
       await assertRevert(repayKUSDPromise_B, "revert");
     });
 
+    it("adjustTrove(): not minting new KUSDToken if mint cap is reached", async () => {
```
Comment (udosson) [inline packages/contracts/test/BorrowerOperationsTest.js:4718]: @arthurka-o - What is the difference between this test and "withdrawKUSD(): not minting new KUSDToken if mint cap is reached"?

Comment (arthurka-o) [reply packages/contracts/test/BorrowerOperationsTest.js:4718]: Even if `withdrawKUSD` and `adjustTrove` are similar, they're still separate external functions. If, in the future, one of the functions adds more/changes its implementation, we will test them separately.

**Review by asimpk — APPROVED** [review]

**Comment by arthurka-o** [comment]
BorrowerOperations > withdrawKUSD --> not minting new KUSDToken if the mint cap is reached
BorrowerOperations > adjustTrove --> not minting new KUSDToken if the mint cap is reached
BorrowerOperations > openTrove --> not minting new KUSDToken if the mint cap is reached
BorrowerOperations > KUSDMintRemainder --> returns the correct value

@udosson

**Comment by udosson** [comment]
> 

Thanks, @arthurka-o 
Can you please add the following test cases for the mint cap:
- Check that the mint remainder value is returned correctly (Check first, adjust mint cap & check again, mint KUSD and check again, repay KUSD and check again)
- Check that the mint cap includes the borrowing fee
- Check that adjusting of the mint cap is working: (Set mint cap to initial value, mint KUSD, decrease the mint cap to below the minted KUSD per asset --> minting should fail, increase mint cap again --> minting should work again)

**Comment by arthurka-o** [comment]
@udosson 

- Added a test
- Already in tests
- Added a test

**Pull request merged**